src/Model.jl

```diff
@@ -4,6 +4,9 @@
 # Contains the geometry information, but no discretization parameters.
 # The exact model used is defined by the list of terms.
 struct Model{T <: Real}
+    # Human-readable name for the model (like LDA, PBE, ...)
+    name::String
```

Comment:
Hm, not so sure about this. A Model is both a mathematical model (functional form of the terms) as well as a specification of the geometry / atomic constitution. This field seems to specialize to just one of these things

Reply:
Yes true, I agree.

On the other hand this makes the short one-line printing of `Model` more meaningful without adding too much special casing depending on which terms are part of the model, that's why I did it. If you have a better idea how to easily achieve this, I'm happy to remove that field.